anta/custom_types.py

```diff
@@ -167,3 +167,33 @@ def validate_regex(value: str) -> str:
 Hostname = Annotated[str, Field(pattern=REGEXP_TYPE_HOSTNAME)]
 Port = Annotated[int, Field(ge=1, le=65535)]
 RegexString = Annotated[str, AfterValidator(validate_regex)]
+BgpDropStats = Literal[
```

Comment:
How did you get that list? Counters look to be different on various platforms. Here is an example on a 7280CR2K:
```
"dropStats": {
  "inDropAsloop": 0,
  "inDropClusterIdLoop": 0,
  "inDropMalformedMpbgp": 0,
  "inDropOrigId": 0,
  "inDropNhLocal": 0,
  "inDropNhAfV6": 0,
  "prefixDroppedMartianV4": 0,
  "prefixDroppedMaxRouteLimitViolatedV4": 0,
  "prefixDroppedMartianV6": 0,
  "prefixDroppedMaxRouteLimitViolatedV6": 0,
  "prefixLuDroppedV4": 0,
  "prefixLuDroppedMartianV4": 0,
  "prefixLuDroppedMaxRouteLimitViolatedV4": 0,
  "prefixLuDroppedV6": 0,
  "prefixEvpnDroppedMaxRouteLimitViolated": 0,
  "prefixRtMembershipDroppedLocalAsReject": 0,
  "prefixRtMembershipDroppedMaxRouteLimitViolated": 0
},
```

Reply:
As discussed added check for verifying all the drop stats and if any mentioned drop stat not found.